### DISCUSSION TITLE
Criteria for labeling papers

### DISCUSSION THREAD

**Discussion by JJMFreriksen** [asreview/asreview#1036] [Q&A] [answered]
For the project with the Dutch Pediatric Formulary, we've created a workflow / protocol with specific criteria to "objectively" label the papers either relevant or irrelevant for establishing a dosing recommendation for a specific drug in children. See attachment. 

**1) Question related to labeling a paper irrelevant while content in title abstract is relevant**
We, however, do now feel that it's perhaps not okay to have "correct language" and "full text available" as criteria. If we, for instance, label Spanish papers as irrelevant, or papers for which the full text is not available, then we train ASReview wrongly, right? Based on title/abstract content, these papers _are_ relevant! But we don't want them. Do you agree that we delete these criteria from the protocol (=label these papers as relevant) and remove them from the dataset _after_ screening?

The same holds true for case reports. These types of papers are considered to be 'low level of evidence' in this field and if 'higher level of evidence' studies, such as large randomized controlled trials with many many subjects are available, we're basically not that interested anymore in the case reports. However, we can't label case reports are irrelevant as we will train ASReview wrongly then. 

**2) Question related to labeling a paper irrelevant/relevant based on information in the full text** 
Also, we only want to have papers in the final dataset which do report the dose explicitly (e.g. a certain amount of milligram). This information is, however, not always provided in the title/abstract so the full text needs to be screened for this information. Can we label a paper based on information in the full text? I think we should delete this crteria from the protocol and do this check (if a dose is given in the full text) after screening. 

Curious to hear your opinion! Thanks.

**Comment by FelixWdm** [comment, marked as answer]
Hi @JJMFreriksen , 

Thanks for sharing these questions. 

1. How to act differs case by case, but for this scenario, we do agree that you should not use the language and full text availability as criteria during the screening using ASReview. ASReview can handle multiple languages ([link](https://github.com/asreview/asreview-multilingual-feature-extractor)), but that's only useful if it's really a mix of languages. In this case I expect the majority will be English with an occasional non-English one. Since the langauge field is available and reliable in most databases, you could also filter out the non-English content during the export. And either discard them completely or look at them seperately.
The distinction between case reports and trials could also be made during the search phase using a [search block](https://blocks.bmi-online.nl/catalog/401), but this would remove all case reports fully from the dataset. It's difficult to predict what happens when you label them as irrelevant, is the set is made up of just relevant trials and irrelevant case reports, you can discard them, but if not, your suggestion to label them relevant seems more 'safe'. 
2. The tradional process of screening also has a title/abstract phase and a full-text phase. The most reliable way is to use ASReview to do the title/abstract screening much faster, not to skip the full-text phase completely. Using ASReview makes that you can collect more results during the search phase, and go through the title/abstract screening much faster. 

I couldn't get the attachment from GitHub, but in general I believe the criteria during the screening phase should be chosen so that you quickly have a much smaller set of mostly relevant papers left. During the next phase you'll have more detailed criteria and can still discard results based on quality etc..

I hope this helps, if not, let us know.

**Reply by JJMFreriksen** [reply]
Thanks @FelixWdm for your clear answer! This really helps and we can now start screening.
